examples/workflows/k8s-resource-log-selector.yaml

```diff
@@ -0,0 +1,48 @@
+# This example demonstrates how to include and obtain logs from pods created by a
```

Comment:
@flaviuvadan since this is an upstream example, could you move the python file to `examples/workflow/upstream`? Then if you run `make regenerate-test-data` it should generate all required yaml files.